Command/DumpCommand.php

```diff
@@ -29,7 +29,27 @@ protected function configure()
                     'Override the target directory to dump JS translation files in.'
                 ),
             ))
-            ->setDescription('Dumps all JS translation files to the filesystem');
+            ->setDescription('Dumps all JS translation files to the filesystem')
+            ->addOption(
+                'locales',
+                null,
+                InputOption::VALUE_REQUIRED | InputOption::VALUE_IS_ARRAY,
+                'If set, only the passed locales will be generated',
+                array()
+            )
+            ->addOption(
+                'formats',
```

Comment:
if you use `` InputOption::VALUE_IS_ARRAY``, you should use a singular name, as the usage for multiple formats will be ``--format=js --format=json``